pkg/config/config.go

```diff
@@ -1126,3 +1134,20 @@ func (c *Config) ActiveDestination() (uri, identity string, err error) {
 	}
 	return "", "", errors.New("no service destination configured")
 }
+
+// ImageCopyTmpDir default directory to store tempory image files during copy
+func (c *Config) ImageCopyTmpDir() (string, error) {
+	path := func() string {
+		if path, found := os.LookupEnv("TMPDIR"); found {
+			return path
+		}
+		if c.Engine.ImageCopyTmpDir == "storage" {
+			return filepath.Join(c.Engine.graphRoot, "tmp")
+		}
+		return c.Engine.ImageCopyTmpDir
+	}()
+	if path[0] != '/' {
```

Comment:
Doesn’t this crash on empty paths? `filepath.IsAbs` would even work on Windows (well, work… it accepts `LPT1` for some reason. But that’s not much of a concern for us.)